src/main/java/apoc/math/Regression.java

```diff
@@ -0,0 +1,63 @@
+package apoc.math;
+
+import org.apache.commons.math3.stat.regression.SimpleRegression;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.ResourceIterator;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Mode;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.util.stream.Stream;
+
+/**
+ * @author <a href="mailto:ali.arslan@rwth-aachen.de">AliArslan</a>
+ */
+public class Regression {
+    @Context
+    public GraphDatabaseService db;
+
+    // Result class
+    public static class Output {
+        public Number r2;
+        public Number avg_x;
+        public Number avg_y;
+        public Number slope;
+        public Output(Number r2, Number avg_x, Number avg_y, Number slope){
+            this.r2 = r2;
+            this.avg_x = avg_x;
+            this.avg_y = avg_y;
+            this.slope = slope;
+        }
+    }
+
+    @Procedure(name = "apoc.math.regr", mode = Mode.READ)
+    public Stream<Output> regr(@Name("label") String label,
+                               @Name("property_y") String y, @Name("property_x") String x ) {
+
+        SimpleRegression regr = new SimpleRegression(false);
+        Number regr_avgx = 0;
```

Comment:
see this [https://github.com/ArslanAle/neo4j-apoc-procedures/commit/8afd7680ce0d064ea94903f8844a196fb6670cf2](commit)